Gems/PhysX/Code/Editor/ColliderCylinderMode.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Editor/ColliderCylinderMode.h>
+
+#include <AzToolsFramework/ComponentModes/BaseShapeComponentMode.h>
+#include <AzToolsFramework/ViewportSelection/EditorSelectionUtil.h>
+#include <PhysX/EditorColliderComponentRequestBus.h>
+
+namespace PhysX
+{
+    AZ_CLASS_ALLOCATOR_IMPL(ColliderCylinderMode, AZ::SystemAllocator, 0);
```

Comment:
FYI there is a [PR](https://github.com/o3de/o3de/pull/14530) going on removing the last parameter in all the codebase, it'd be nice not to include the third parameter here :)

Reply:
Done